src/app/features/ahbs/components/ahb-table/ahb-table.component.spec.ts

```diff
@@ -1,22 +1,13 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { AhbTableComponent } from './ahb-table.component';
+import { MockBuilder, MockRender, ngMocks } from 'ng-mocks';
```

Comment:
I know mocks only from unit tests. But these lines look like production code.
What is the purpose of the MockBuilder here?

Reply:
files that end in `.spec.ts` are unit tests.